packages/babel-parser/test/fixtures/jsx/errors/_no-plugin-type-param/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "plugins": [],
```

Comment:
Can you add `jsx` here? I think it will still throw syntax error.

Reply:
It does throw that error 

```
SyntaxError: Unexpected token (1:13)
    > 1 | <div>() => {}
        |              ^
```

What would the correct error message (if any) be in that case?

Reply:
If we can throw something like "expected closing JSX element" it's better, otherwise `Unexpected token` is ok for now.

Reply:
Sounds good to me, pushing that final test